### DISCUSSION TITLE
Welcome & introductions!

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#1] [Miscellaneous]
## Welcome!
  We’re using Discussions as a place to discuss Mathesar's architecture and roadmap connect with members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mathemancer** [comment]
Hello everyone,

I'm Brent, a data engineer.  I worked with @kgodey previously at Creative Commons on CC Search.  I'm working now as the data engineer on the new Mathesar project.

**Comment by dhruvkb** [comment]
Hello,

I'm Dhruv. I worked with @kgodey and @mathemancer at Creative Commons, where I was a community maintainer for [CC Vocabulary][vocabulary], the frontend library used in CC Search. I'll switch between doing some backend and some frontend work on Mathesar!

[vocabulary]: https://github.com/creativecommons/cc-vocabulary

**Comment by jbruni** [comment]
Hi there,

I've been coding since I was a kid and wanted to create my own games for others to play, using MSX-BASIC back in 1990. Never dived too deep in complexity because web development allows you to do so many things without the need of advanced programming skills. But, eventually, along the way and the years, I acquired some experience from solving one problem after the other. 😄 

I am not sure how my Mathesar participation will be, but I've already started engaging and sharing ideas. 💡 

Thank you, all!
